keras_core/backend/jax/image.py

```diff
@@ -40,3 +45,112 @@ def resize(
             f"image.shape={image.shape}"
         )
     return jax.image.resize(image, size, method=method, antialias=antialias)
+
+
+AFFINE_METHODS = {  # map to order
+    "nearest": 0,
+    "bilinear": 1,
+}
+AFFINE_FILL_MODES = (
+    "constant",
+    "nearest",
+    "wrap",
+    "mirror",
+    "reflect",
+)
+
+
+def affine(
+    image,
+    transform,
+    method="bilinear",
+    fill_mode="constant",
+    fill_value=0,
+    data_format="channels_last",
+):
+    if method not in AFFINE_METHODS.keys():
+        raise ValueError(
+            "Invalid value for argument `method`. Expected of one "
+            f"{AFFINE_METHODS.keys()}. Received: method={method}"
```

Comment:
This will print "dict_keys()". Let's cast to `set()` first.

Reply:
Done for all backends.